### DISCUSSION TITLE
Nativescript Dark Mode Issue: App Converts into Dark Mode even if it is made for Light Mode only in Android

### DISCUSSION THREAD

**Discussion by abhishekranwa** [NativeScript/NativeScript#9812] [Q&A]
### Issue Description

This issue is happening only on Android. The App is Only made for Light Mode but when a user clicks on the Dark mode button from the dropdown the app also converts into Dark Mode. We want to disable the dark mode in the App. The App Should only be in Light Mode even if someone Presses the Dark Mode button from the Dropdown. Please see the screenshot for a better understanding of the issue.

### Reproduction

**To Reproduce**
Open the App in android and the app is in light mode, but when the user clicks on the dark mode from the dropdown the app converts into dark mode.

**Expected behavior**
The App should always be in Light Mode.

**Error screenshot image**
**_In Light Mode:_**
![1f99f87c-3da7-47a2-a522-2664512c7719](https://user-images.githubusercontent.com/92712353/152329112-e40a4ec6-1bf3-4563-8131-2e4bbac270db.JPG)

**_In Dark Mode:_**
![3454b130-7b8b-488d-a419-dd0fa7a82ac4](https://user-images.githubusercontent.com/92712353/152329169-a2b229f0-884d-4758-9b81-1a8a2ae55d70.JPG)

### Environment

**Environment :**
 --> tns info
✔ Getting NativeScript components versions information...
⚠ Update available for component native script. Your current version is 7.2.0 and the latest available version is 8.1.5.
⚠ Update available for component @nativescript/core. Your current version is 7.0.13 and the latest available version is 8.1.5.
✔ Component @nativescript/ios has an 8.1.0 version and is up to date.
✔ Component tns-android has a 6.5.3 version and is up to date.
    "@nativescript/theme": "~2.3.0"

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by vallemar** [comment]
@abhishekranwa  Try this on main.js or TS
```ts
import Theme from "@nativescript/theme";

Theme.setMode(Theme.Light);
```

Reference: https://stackoverflow.com/questions/59638458/how-to-effectively-disable-dark-mode-theming-in-nativescript

**Comment by abhishekranwa** [comment]
@vallemar  Already did this. But after opening the App when the user clicks on the Dark mode from dropdown, App changes into dark mode.

Tried these steps from: 
https://medium.com/@kivind/nativescript-disabling-dark-mode-382e5dfd11bd

But still the same issue

**Comment by rigor789** [comment]
I have a draft doc about some settings in App_Resources, I believe you just need to set 

```xml
<item name="android:forceDarkAllowed">false</item>
```

to prevent the default android dark mode being forced.

https://github.com/NativeScript/docs-new/blob/49b26cce67aa3d7463ec2187db49ad995d44d6a1/app-resources.md#enabling-force-dark-mode

**Reply by abhishekranwa** [reply]
@rigor789 I already have this in my App_Resources, but the app is still going in dark mode.
How can I prevent it from going to dark mode in Android?

**Comment by abdallahkadour** [comment]
I have also the same issue, any update on this?

**Reply by abhishekranwa** [reply]
No 
Still the same issue

**Comment by abdallahkadour** [comment]
I fixed the issue by removing the imported themes in app.scss.
`//@import "@nativescript/theme/css/core.css";
//@import "@nativescript/theme/css/default.css";`

**Comment by jitendraP-ashutec** [comment]
@rigor789  
There is still issue to disable dark mode in android. I have tried in nativescript 8.5.0. But its seems it not working. I have tried all solution suggested in here: https://medium.com/@kivind/nativescript-disabling-dark-mode-382e5dfd11bd.

Its apply to disable theme, but after if user make some manual events. Like user go to another page and come back then its working as expected

**Reply by vallemar** [reply]
removing the @nativescript/theme package, I think this fixes it
